backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceHeaderImpl.java

```diff
@@ -400,4 +404,27 @@ public SectionCodeId getM_SectionCode_ID()
 	{
 		return sectionCodeId;
 	}
+
+	@Nullable
+	@Override
+	public String getInvoiceAdditionalText()
+	{
+		return invoiceAdditionalText;
+	}
+
+	@Override
+	public boolean isNotShowOriginCountry()
+	{
+		return notShowOriginCountry;
+	}
+
+	public void setInvoiceAdditionalText(final String invoiceAdditionalText)
+	{
+		this.invoiceAdditionalText = invoiceAdditionalText;
+	}
+
+	public void setNotShowOriginCountry(final boolean notShowOriginCountry)
+	{
+		this.notShowOriginCountry = notShowOriginCountry;
+	}
```

Comment:
Why is this here? You are not aggregating by this field.

Reply:
To copy the new fields from Invoice candidate to the invoice

Reply:
What I meant is that, because you're not ensuring uniqueness of the flag for all ICs (via aggregation), it's always taken from the first/last processed IC.
Is this intended?

Reply:
@riadmen (@adi-stefan ) right, i also had this concern in the TC review, but then wrote that you are handling it further down the TC. But unfortunately just just do it in the user's custom intance which is not sufficient.
you need to add `InvoiceAdditionalText` to the *standard* metasfresh's `invoicing-agg-std` aggregator.

and with respect to `notShowOriginCountry` =>if this is going into C_Invoice (and not into C_ÌnvoiceLine), then it *also* needs to go into the *standard* metasfresh's `invoicing-agg-std` aggregator.